android/app/src/main/java/org/fossasia/openevent/fragments/FeedFragment.java

```diff
@@ -116,7 +116,7 @@ public void handleVisibility() {
     @Override
     public void onDestroyView() {
         super.onDestroyView();
-
+        feedAdapter.setOnImageZoomListener(null);
```

Comment:
Actually feedAdapter has a method called ```removeOnImageZoomListener```. Also it is already implemented inside the ```onStop```, so you don't need this.

Reply:
Oh yeah true! Thanks :)